test/engine_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule EngineTest do
+  use ExUnit.Case
+
+  test "list_spiders/0 lists all spiders and their current status in the engine" do
+    assert spiders = Crawly.Engine.list_spiders()
+    assert [_ | _] = spiders
```

Comment:
What happens here? Just in case I am a bit worried about the case when we add more test spiders... Will this test =fail automatically in this case?

Reply:
I'll remove the unnecessary assertion on line 5, it should never fail.